eagle-jpm/eagle-jpm-util/src/main/java/org/apache/eagle/jpm/util/jobrecover/RunningJobManager.java

```diff
@@ -49,12 +53,21 @@ private CuratorFramework newCurator(String zkQuorum, int zkSessionTimeoutMs, int
         );
     }
 
-    public RunningJobManager(String zkQuorum, int zkSessionTimeoutMs, int zkRetryTimes, int zkRetryInterval, String zkRoot) {
+    public RunningJobManager(String zkQuorum, int zkSessionTimeoutMs, int zkRetryTimes, int zkRetryInterval, String zkRoot, String siteId) {
```

Comment:
OK. I will generate lockpath in utility class.Then RunningJobManager will use it. Lockpath format is /apps/{app}/{site}/locks.